Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/InputValue.cs

```diff
@@ -45,6 +82,36 @@ public TValue Get<TValue>()
         }
 
         ////TODO: proper message if value type isn't right
+        /// <summary>
+        /// Check if the action button is pressed.
+        /// </summary>
+        /// <returns>True if the button is activated over the button threshold. False otherwise</returns>
+        /// <remarks>
+        /// The following example shows how to read a value from a <see cref="PlayerInput"/> message.
```

Comment:
Minor: I think message is odd here- though technically correct. action event? Or: "retrieving and utilizing the InputValue from an input action event." ?

Reply:
I need to make it clear this isn't an event as those are different to messages. I've adjusted a little to mention notification messages.